dlt/dlt.py

```diff
@@ -22,7 +22,7 @@
     DLT_TYPE_INFO_UINT, DLT_TYPE_INFO_SINT, DLT_TYPE_INFO_STRG, DLT_TYPE_INFO_SCOD,
     DLT_TYPE_INFO_TYLE, DLT_TYPE_INFO_VARI, DLT_TYPE_INFO_RAWD,
     DLT_SCOD_ASCII, DLT_SCOD_UTF8, DLT_TYLE_8BIT, DLT_TYLE_16BIT, DLT_TYLE_32BIT, DLT_TYLE_64BIT,
-    DLT_TYLE_128BIT, DLT_DAEMON_TCP_PORT, DLT_CLIENT_RCVBUFSIZE,
+    DLT_TYLE_128BIT, DLT_DAEMON_TCP_PORT, DLT_RECEIVE_BUFSIZE,
```

Comment:
I can not find the variable `DLT_RECEIVE_BUFSIZE ` on the original repo or your patch